example/lib/VisionDetectorViews/painters/pose_painter.dart

```diff
@@ -29,6 +29,70 @@ class PosePainter extends CustomPainter {
             1,
             paint);
       });
+
+      final leftPaint = Paint()
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 6
```

Comment:
make the strokeWidth no greater than the strokeWidth of the paint used for the dots

Reply:
Okay, I will make the changes in accordance with your reviews. Thank you!